src/main/scala/com/mpc/scalats/core/ScalaModel.scala

```diff
@@ -5,25 +5,37 @@ import scala.collection.immutable.ListSet
 // TODO: ValueClass
 
 object ScalaModel {
+  case class QualifiedIdentifier(
+    name: String,
+    enclosingClassNames: List[String]
```

Comment:
Why `List`?

Reply:
To store names of the entire chain of enclosing classes, e.g. for:

```scala
class A { class B { class C } } }
```

we're gonna have `QualifiedIdentifier(name="C", enclosingClassNames=List("A", "B"))`

Reply:
Then maybe either `type QualifiedIdentifier = (String, List[String], String)` or `class QualifiedIdentifier(val root: String, parent: List[String], child: String)`.

`List[String]` doesn't offer a guarantee to have any parents.

Reply:
Hmmm but a situation when there are no parents at all is also perfectly acceptable, for instance  a plain `class C` will correspond just to `QualifiedIdentifier(name="C", enclosingClassNames=Nil)`